exdb/forms.py

```diff
@@ -152,7 +153,8 @@ class Meta:
 
     def clean(self):
         conditions = (
-            (not self.cleaned_data.get('attendance'), ValidationError(_('There must be an attendance'))),
+            (not self.cleaned_data.get('attendance') and self.cleaned_data.get('attendance', -1) != 0,
```

Comment:
Alternately, this could've been changed to
```python
(self.cleaned_data.get('attendance', -1) < 0, ValidationError(_('There must be an attendance'))),
```